integration/setup/listener.go

```diff
@@ -81,6 +83,10 @@ func listenerMongoDBURI(tb testtb.TB, hostPort, unixSocketPath string, tlsAndAut
 		user = url.UserPassword("username", "password")
 	}
 
+	if enableNewAuth {
+		q.Set("authMechanism", "SCRAM-SHA-256")
```

Comment:
So, in MongoDB we always set `SCRAM-SHA-256`, but in FerretDB tests we sometimes use `SHA-1` too, right? Does it cause any problems with tests?

Reply:
FerretDB uses `SHA-256` for new authentication, `PLAIN` old authentication. Both FerretDB and MongoDB tests `SHA-1` in specific integration tests by creating user and creating client with `SHA-1`. I don't see a problem :thinking:

Reply:
> So, in MongoDB we always set `SCRAM-SHA-256`, but in FerretDB tests we sometimes use `SHA-1` too, right?

This code configures in-process FerretDB with either `PLAIN` or `SCRAM-SHA-256`